test/testlib/NuoDBRegistryEntry.go

```diff
@@ -15,10 +15,8 @@ type Registry struct {
 }
 
 // UnmarshalImageYAML is used to unmarshal into map[string]string
-func UnmarshalImageYAML(s string) (err error, registry Registry) {
-	registry = Registry{}
-
-	err = yaml.Unmarshal([]byte(s), &registry)
-
-	return
```

Comment:
change makes sense here, wondering how this worked before without returning anything?

Reply:
Previously the return types had the local variables `err` and `registry` associated with them, which were specified on the function signature. So simply setting these variables was enough to generate the correct return value. But this is a weird syntax that we don't use anymore.